pkg/sql/schemachanger/scdecomp/decomp.go

```diff
@@ -488,13 +493,29 @@ func (w *walkCtx) walkIndex(tbl catalog.TableDescriptor, idx catalog.Index) {
 			IsNotVisible:        idx.IsNotVisible(),
 		}
 		for i, c := range cpy.KeyColumnIDs {
+			implicitColumn := false
+			if i < idx.ImplicitPartitioningColumnCount() {
+				implicitColumn = true
+			}
+			invertedKind := scpb.IndexColumn_NOT_INVERTED
+			if index.IsInverted && c == idx.InvertedColumnID() {
+				invertedKind = scpb.IndexColumn_INVERTED
+				switch idx.InvertedColumnKind() {
+				case catpb.InvertedIndexColumnKind_DEFAULT:
+					invertedKind = scpb.IndexColumn_INVERTED
+				case catpb.InvertedIndexColumnKind_TRIGRAM:
+					invertedKind = scpb.IndexColumn_INVERTED_TRIGRAM
+				}
+			}
```

Comment:
Why not use the existing catpb enum directly? What purpose does this scpb enum serve?